src/main/java/Listmanager.java

```diff
@@ -0,0 +1,133 @@
+import java.util.Scanner;
+
+public class Todolist {
+
+    String Icon = "T";
+    private static final int MAX_LIST_LENGTH = 100;
+    static String[] todolist = new String[MAX_LIST_LENGTH];
+    static boolean[] isDoneList = new boolean[MAX_LIST_LENGTH];
+
+    static int ListCount= 0;
+    static Scanner in = new Scanner(System.in);
+
+    public static void AddToList(){
+        String Task= in.nextLine();
+        todolist[ListCount] = Task;
+        isDoneList[ListCount]= false;
+        ListCount++;
+        return;
+    }
+
+
+
+    static void HandleList(){
+        PrintList();
+        String command ="";
+
+        while(!command.equals("quit")){
+            command = in.nextLine();
+            switch (command){
+            case"add":
+                AddToList();
+                System.out.println("Task added");
+                PrintList();
+                break;
+            case"quit":
+                System.out.println("Task change complete");
+                break;
+            case"help":
+                System.out.print("supported commands: add , quit , help , mark,unmark ,clear, list\n");
+                break;
+            case "clear":
+                ClearList();
+                ClearisDoneList();
+                System.out.print("list cleared\n");
+                break;
+            case"mark":
+                SetDone();
+                break;
+            case"unmark":
+                SetUndone();
+                break;
+            case"list":
+                PrintList();
+                break;
+            case"":
+                break;
+            default:
+                System.out.println("Sorry, I can not understand this command, enter \"help\" for help\n");
+                break;
+            }
+
+        }
+
+    }
+
+    public static void PrintList(){
+        int i=1;
+        System.out.print("---------------------------------------------\n");
```

Comment:
Consider making this a constant to prevent repetition of the same code and will be more consistent when modification is made